src/main/java/com/github/spring/jwt/JwtCrackerApplication.java

```diff
@@ -75,7 +77,14 @@ public void run(String... args) throws Exception {
             LOG.info("total time taken [hh::mm:ss:SSS] {}", watch);
             LOG.info("************************************************");
         }
+        
+        MyClosable cl = new MyClosable(new ByteArrayOutputStream());
+        cl.write("hello");
     }
+    
+   /* private void call(MyClosable cl) throws IOException {

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)